samples/automl/automlTranslationDataset.js

```diff
@@ -60,22 +60,24 @@ function createDataset(projectId, computeRegion, datasetName, source, target) {
       const dataset = responses[0];
 
       // Display the dataset information
-      console.log('Dataset name:', dataset.name);
-      console.log('Dataset id: ', dataset.name.split('/').pop(-1));
-      console.log('Dataset display name: ', dataset.displayName);
-      console.log('Dataset example count: ', dataset.exampleCount);
-      console.log('Translation dataset specification:');
+      console.log(`Dataset name: ${dataset.name}`);
+      console.log(`Dataset id: ${dataset.name.split(`/`).pop(-1)}`);
+      console.log(`Dataset display name: ${dataset.displayName}`);
+      console.log(`Dataset example count: ${dataset.exampleCount}`);
+      console.log(`Translation dataset specification:`);
       console.log(
-        '\tSource language code: ',
-        dataset.translationDatasetMetadata.sourceLanguageCode
+        `\tSource language code: ${

```

Comment:
Nit: separate lines are fine for multi-line `console.log` statements.

Apply this comment throughout this PR.